components/channel/components/native.ts

```
@@ -0,0 +1,427 @@
+}) => Promise<Photo> | never;
+export let takePhoto: TakePhoto = fail;
+
+type HapticFeedbackMethod =
+  | "impactHeavy"
+  | "impactLight"
+  | "impactMedium"
+  | "notificationError"
+  | "notificationSuccess"
+  | "notificationWarning"
+  | "selection";
+type TriggerHaptic = (method: HapticFeedbackMethod) => void | never;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)